dasharo-performance/cpu-temperature.robot

```diff
@@ -124,15 +124,24 @@ CPU Temperature Without Load (Ubuntu 22.04)
     Detect Or Install Package    lm-sensors
     Execute Command In Terminal    sensors-detect --auto
     ${timer}=    Convert To Integer    0
+    ${temperatures}=    Create List
+    ${max_temperature}=    Set Variable    0
     FOR    ${i}    IN RANGE    (${TEMPERATURE_TEST_DURATION} / ${TEMPERATURE_TEST_MEASURE_INTERVAL}) + 1
         Log To Console    \n ----------------------------------------------------------------
         Log To Console    ${timer} min.
         ${temperature}=    Get CPU Temperature CURRENT
         Log To Console    Current temperature: ${temperature}°C
-        Should Be True    ${temperature} < ${MAX_CPU_TEMP}
+        ${temperature}=    Evaluate    float(${temperature})
+        Append To List    ${temperatures}    ${temperature}
+        ${max_temperature}=    Evaluate    max(${max_temperature}, ${temperature})
         Sleep    ${TEMPERATURE_TEST_MEASURE_INTERVAL}m
         ${timer}=    Evaluate    ${timer} + ${TEMPERATURE_TEST_MEASURE_INTERVAL}
     END
+    ${mean_temperature}=    Evaluate    sum(${temperatures}) / len(${temperatures})
+    Log To Console    Mean temperature over test duration: ${mean_temperature}°C
```

Comment:
[`Log To Console` keyword shouln't be used to write to terminal](https://github.com/Dasharo/open-source-firmware-validation/blob/main/README.md?plain=1#L299-L307). Applies to all instances

Reply:
My bad. Log to console is used correctly here. The mentioned keywords are used for writing to the platform's terminal remotely